cli/cmd/plugin/unmanaged-cluster/log/log_test.go

```diff
@@ -0,0 +1,41 @@
+package log
```

Comment:
For some reason the linter output isn't showing it, but the issue is this file needs the standard copyright/license header. You should be able to just copy the header from another file.

Reply:
Yes sir. I'm in the process of doing that.. Missed it the first time around

Reply:
Changes have been updated. @stmcginnis  PTAL